### PR TITLE
feat(console): sie sign up form

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

- remove the `Sign In Methods` tab
- add the `Sign Up And Sign In` tab
- add the sign-up form

## Testing

sign-up-form

### CODE REVIEW

File: packages/phrases/src/locales/en/translation/admin-console/sign-in-exp.ts [hunk 1]
```diff
@@ -38,6 +38,22 @@ const sign_in_exp = {
     slogan: 'Slogan',
     slogan_placeholder: 'Unleash your creativity',
   },
+  sign_up_and_sign_in: {
+    identifiers: 'Sign up identifiers', // UNTRANSLATED
```
Comment (wangsijie) [inline packages/phrases/src/locales/en/translation/admin-console/sign-in-exp.ts:42]: Is the "UNTRANSLATED" tag indicates that this is not reviewd by @BTX26?

File: packages/console/src/pages/SignInExperience/tabs/SignUpAndSignInTab.tsx [hunk 2]
```diff
@@ -22,10 +22,10 @@ const SignInMethodsTab = ({ defaultData, isDataDirty }: Props) => {
 
   return (
     <>
-      <SignInMethodsForm />
+      <SignUpAndSignInForm />
```
Comment (wangsijie) [inline packages/console/src/pages/SignInExperience/tabs/SignUpAndSignInTab.tsx:25]: Can we seperate this into several forms, as we did before, every section(have a title in Figma) has a form.

File: packages/phrases/src/locales/en/translation/admin-console/sign-in-exp.ts (hunk 1 above)
Comment (xiaoyijun) [reply packages/phrases/src/locales/en/translation/admin-console/sign-in-exp.ts:42]: Yes, It is also used to mark the newly added but not determined translations.

File: packages/console/src/pages/SignInExperience/tabs/SignUpAndSignInTab.tsx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/tabs/SignUpAndSignInTab.tsx:25]: We do this as we did before.
Just replace the `SignInMethodsForm` with the new one, and they are equivalent in structure.

File: packages/phrases/src/locales/en/translation/admin-console/sign-in-exp.ts (hunk 1 above)
Comment (xiaoyijun) [reply packages/phrases/src/locales/en/translation/admin-console/sign-in-exp.ts:42]: Oh, this field in `en` is determined; Tag removed.

File: packages/console/src/pages/SignInExperience/tabs/SignUpAndSignInTab.tsx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/pages/SignInExperience/tabs/SignUpAndSignInTab.tsx]: Reorged.

**Review by wangsijie — APPROVED** [review]
LGTM

**Pull request merged**